keyboards/murcielago/rev1/config.h

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2020 elagil
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0x0000
+#define PRODUCT_ID 0x0001
+#define DEVICE_VER 0x0002
+#define MANUFACTURER elagil
+#define PRODUCT murcielago
+#define DESCRIPTION A custom keyboard with 60 keys
+
+/* key matrix size */
+#define MATRIX_ROWS 12
+#define MATRIX_COLS 6
+
+/* split options, use EEPROM for side detection */
+#define EE_HANDS
+#define SPLIT_USB_DETECT
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { B4, D5, B3, B2, B1, B0 }
+#define MATRIX_COL_PINS { C6, B6, B5, D7, D6, D4 }
+#define MATRIX_ROW_PINS_RIGHT { B3, B7, D4, B6, C6, C7 }
+#define MATRIX_COL_PINS_RIGHT { D6, D7, D3, B2, B1, B0 }
+
+#define ENCODERS_PAD_A { F0 }
+#define ENCODERS_PAD_B { F1 }
+#define ENCODERS_PAD_A_RIGHT { F6 }
+#define ENCODERS_PAD_B_RIGHT { F7 }
+
+#define BACKLIGHT_PIN B7
+#define BACKLIGHT_PIN_RIGHT B5
```

Comment:
```suggestion
```

Reply:
I added backlight support for split boards in #8828 
Should I remove this until the other PR is merged or just leave it in?

Reply:
TBH, that PR introduces a lot of duplicate code, and I don't think that is the best way to handle it. Given how backlight config is setup right now it may not be practical to support different pins on each half. Hopefully some refactoring can help with that, but I'm not 100% sure.

Reply:
Yes, I agree that it is not very clean. I will try to think of a more elegant way.

Reply:
My main problem is that the backlight driver uses defines, so I cannot change anything at runtime. However, this is required after detection of handedness.